### PR TITLE
Create dublinCA

### PR DESCRIPTION
**Pull request by sureshkg**
Testing pull request for Restaurants

### CODE REVIEW

File: sureshkg/dublinCA [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Restaurtants in Dublin CA
+
+### 
+
```
Comment (ArthurZC23) [inline sureshkg/dublinCA:4]: @sureshkg  which one is the best? :smiley:

**Comment by rwim2** [comment]
Any great places in Dublin you like to eat. I am in Fremont

**Comment by bretgithub** [comment]
@sureshkg 

What is your favorite restaurant in Dublin?

File: sureshkg/dublinCA [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Restaurtants in Dublin CA
```
Comment (it01-GlacierWater) [inline sureshkg/dublinCA:1]: @sureshkg Do they have a Pizza My Heart in Dublin?

File: sureshkg/dublinCA (hunk 1 above)
Comment (yangus1265) [inline sureshkg/dublinCA:4]: @sureshkg Bawarchi sounds interesting :100:

Comment (sureshkg) [inline sureshkg/dublinCA:4]: @ArthurZC23, Bawarchi is my favorite!

**Pull request merged**